# French translations for PACKAGE package
# Traductions françaises du paquet PACKAGE.
# Copyright (C) 2010 THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# Automatically generated, 2010.
#
msgid ""
msgstr ""
"Project-Id-Version:  vigilo.corr\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2010-04-23 13:54+0200\n"
"PO-Revision-Date: 2010-06-29 18:40+0200\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: fr <LL@li.org>\n"
"Plural-Forms: nplurals=2; plural=(n > 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.4\n"

#: src/vigilo/correlator/compute_hls_states.py:146
#, python-format
msgid "Dependancies have the following weights: %r"
msgstr "Les poids des dépendances sont les suivants : %r"

#: src/vigilo/correlator/compute_hls_states.py:184
#, python-format
msgid ""
"High-level service %(name)r: weight = %(weight)r, state = %(state)r "
"(%(output)r)"
msgstr ""
"Service de haut niveau %(name)r : poids = %(weight)r, état = %(state)r "
"(%(output)r)"

#: src/vigilo/correlator/compute_hls_states.py:236
#, python-format
msgid "End of impact for HLS %r"
msgstr "Le service de haut niveau %r n'est plus impacté."

#: src/vigilo/correlator/compute_hls_states.py:253
#, python-format
msgid "End of impact for item (%(hostname)r, %(servicename)r)"
msgstr "L'item (%(hostname)r, %(servicename)r) n'est plus impacté."

#: src/vigilo/correlator/compute_hls_states.py:275
#, python-format
msgid "HLS %(hls1)r is impacted by HLS %(hls2)r"
msgstr "Le SHN %(hls1)r est impacté par le SHN %(hls2)r"

#: src/vigilo/correlator/compute_hls_states.py:293
#, python-format
msgid ""
"The HLS %r already belonged to this impacted path, ignoring request to "
"add it again."
msgstr ""
"Le SHN %r fait déjà partie de ce chemin d'impact, la requête d'ajout est "
"par conséquent ignorée."

#: src/vigilo/correlator/compute_hls_states.py:322
msgid "Copying path"
msgstr "Copie du chemin"

#: src/vigilo/correlator/compute_hls_states.py:337
#, python-format
msgid "HLS %(hls)r is impacted by item (%(host)r, %(service)r)"
msgstr "Le SHN %(hls)r est impacté par l'item (%(host)r, %(service)r)r"

#: src/vigilo/correlator/compute_hls_states.py:375
#, python-format
msgid "Setting weight for '%s' to None"
msgstr "Changement du poids de '%s' en 'None'"

#: src/vigilo/correlator/correvent.py:112
#, python-format
msgid "Updating existing correlated event (%r)"
msgstr "Mise à jour d'un événement corrélé existant (%r)"

#: src/vigilo/correlator/correvent.py:115
#, python-format
msgid "Got a reference to a non-existent correlated event (%r), adding as new"
msgstr ""
"L'événement corrélé référencé (%r) n'existe pas, il est donc ajouté à \"\n"
"\"la BDD."

#: src/vigilo/correlator/correvent.py:125
#, python-format
msgid "Raw event ignored. Reason: status = %r"
msgstr "Événement brut ignoré. Raison : statut = %r"

#: src/vigilo/correlator/correvent.py:151
#, python-format
msgid ""
"Got a reference to a nonexistent correlated event (%r), skipping this "
"aggregate"
msgstr "L'événement corrélé référencé (%r) n'existe pas, l'agrégat est ignoré."

#: src/vigilo/correlator/correvent.py:194
msgid "Creating new correlated event"
msgstr "Création d'un nouvel événement corrélé"

#: src/vigilo/correlator/correvent.py:308
msgid "Sending correlated event to syslog"
msgstr "Envoi de l'événement corrélé vers le syslog"

#: src/vigilo/correlator/db_insertion.py:44
#, python-format
msgid "Received request to add an event on HLS \"%s\""
msgstr "Réception d'une requête demandant l'ajout d'un événement sur le SHN \"%s\""

#: src/vigilo/correlator/db_insertion.py:52
#, python-format
msgid ""
"Got a reference to a non configured item (%(host)r, %(service)r), "
"skipping event"
msgstr ""
"L'item référencé (%(host)r, %(service)r) n'existe pas, l'événement est "
"ignoré"

#: src/vigilo/correlator/db_insertion.py:85
#, python-format
msgid "insert_event: updating event %r"
msgstr "insert_event : mise à jour de l'événement %r"

#: src/vigilo/correlator/db_insertion.py:91
#, python-format
msgid ""
"Ignoring request to create a new event with state \"%s\" (nothing "
"alarming here)"
msgstr ""
"Demande de création d'un nouvel événement avec l'état \"%s\" ignorée "
"(rien d'alarmant)"

#: src/vigilo/correlator/db_insertion.py:99
#, fuzzy
msgid "insert_event: creating new event"
msgstr "insert_event : création d'un nouvel événement"

#: src/vigilo/correlator/db_insertion.py:102
msgid "Multiple matching events found, skipping."
msgstr "Plusieurs événements concernant l'item ont été trouvés, message ignoré."

#: src/vigilo/correlator/db_insertion.py:128
#: src/vigilo/correlator/db_insertion.py:159
msgid "Got exception"
msgstr "Une exception a été levée"

#: src/vigilo/correlator/db_insertion.py:198
#, fuzzy
msgid "insert_state"
msgstr "insert_state: %r"

#: src/vigilo/correlator/db_insertion.py:218
#, fuzzy
msgid "add_to_aggregate"
msgstr "add_to_aggregate: %r"

#: src/vigilo/correlator/db_insertion.py:235
#, python-format
msgid "Merging aggregate #%(src)d with aggregate #%(dest)d"
msgstr "Fusion de l'agrégat #%(src)d avec l'éagrégat #%(dest)d"

#: src/vigilo/correlator/db_insertion.py:246
msgid ""
"merge_aggregates: Got a reference to a nonexistent source aggregate, "
"aborting"
msgstr "merge_aggregates: L'agrégat source référencé n'existe pas, annulation"

#: src/vigilo/correlator/db_insertion.py:256
#, python-format
msgid ""
"merge_aggregates: Got a reference to a nonexistent destination aggregate "
"%r, aborting"
msgstr ""
"merge_aggregates: L'agrégat destination référencé (%r) n'existe pas, "
"annulation"

#: src/vigilo/correlator/handle_ticket.py:32
#, python-format
msgid ""
"handle_ticket: Trouble ticket message received. Timestamp = "
"%(timestamp)r. Impacted HLS = %(hls)r. Ticket id = %(ticket_id)r. "
"acknowledgement_status = %(ack_status)r. Message = %(message)r."
msgstr ""
"handle_ticket: Message reçu sur un ticket d'incident. Horodatage = "
"%(timestamp)r. SHN impacté = %(hls)r. ID ticket = %(ticket_id)r. état "
"d'acquittement = %(ack_status)r. Message = %(message)r."

#: src/vigilo/correlator/handle_ticket.py:54
#, python-format
msgid "handle_ticket: No matching trouble ticket found : %r"
msgstr "handle_ticket: Aucun ticket d'incident trouvé pour %r"

#: src/vigilo/correlator/handle_ticket.py:60
#, python-format
msgid "handle_ticket: Several events seem to be associated with this ticket : %r"
msgstr "handle_ticket: Plusieurs événements semblent associés à ce ticket : %r"

#: src/vigilo/correlator/handle_ticket.py:63
#, python-format
msgid ""
"handle_ticket: The event %(event_id)r is associated with the given ticket"
" (%(ticket_id)r)"
msgstr ""
"handle_ticket: L'événement %(event_id)r est associé au ticket indiqué "
"(%(ticket_id)r)"

#: src/vigilo/correlator/handle_ticket.py:85
#, python-format
msgid "handle_ticket: Got exception while updating event %r history"
msgstr ""
"handle_ticket: Une exception a été levée durant la mise à jour de "
"l'historique de l'événement %r"

#: src/vigilo/correlator/handle_ticket.py:89
#, python-format
msgid "handle_ticket: Event %r history updated successfully."
msgstr "handle_ticket: L'historique de l'évenement %r a bien été mis à jour"

#: src/vigilo/correlator/memcached_connection.py:167
#, python-format
msgid "Establishing connection to MemcacheD server (%s)..."
msgstr "Connection en cours au serveur MemcacheD (%s) ..."

#: src/vigilo/correlator/memcached_connection.py:189
#, fuzzy, python-format
#| msgid "MemcachedConnection: Trying to set value '%(value)s' for key
#| '%(key)s'..."
msgid "Trying to set value '%(value)s' for key '%(key)s'."
msgstr ""
"MemcachedConnection: Tentative de changement de la valeur de la clé "
"'%(key)s' en '%(value)s'..."

#: src/vigilo/correlator/memcached_connection.py:235
#, fuzzy, python-format
#| msgid "MemcachedConnection: Trying to get the value of the key '%(key)s'..."
msgid "Trying to get the value of the key '%(key)s'."
msgstr ""
"MemcachedConnection: Tentative de récupération de la valeur de la clé "
"'%(key)s'..."

#: src/vigilo/correlator/memcached_connection.py:280
#, fuzzy, python-format
#| msgid "MemcachedConnection: Trying to delete the key '%(key)s'..."
msgid "Trying to delete the key '%(key)s'."
msgstr "MemcachedConnection: Tentative de suppression de la clé '%(key)s'..."

#: src/vigilo/correlator/registry.py:55
#, python-format
msgid "Rule %r has already been registered, ignoring attempt to re-register it."
msgstr ""
"La règle %r a déjà été enregistrée, tentative de ré-enregistrement "
"ignorée."

#: src/vigilo/correlator/registry.py:63
#, python-format
msgid "The rule %(depended)r must be loaded before %(dependent)r."
msgstr "La règle %(depended)r doit être chargée avant  %(dependent)r."

#: src/vigilo/correlator/registry.py:71
#, python-format
msgid "Successfully registered rule %r"
msgstr "Enregistrement de la règle %r réussi"

#: src/vigilo/correlator/registry.py:129
msgid ""
"Singleton has already been instanciated, ignoring attempt to create a new"
" instance."
msgstr ""
"Le singleton a déjà été instancié, demande de création d'une nouvelle "
"instance ignorée."

#: src/vigilo/correlator/twisted_service.py:50
msgid "The topology has been reloaded."
msgstr "L'arbre topologique a été reconstruit."

#: src/vigilo/correlator/twisted_service.py:64
msgid ""
"Could not set signal handlers. The correlator may not be able to shutdown"
" cleanly"
msgstr ""
"Impossible d'installer le gestionnaire de signaux. Le corrélateur risque "
"de ne pas pouvoir être arrêté proprement"

#: src/vigilo/correlator/actors/rule_dispatcher.py:110
msgid ""
"No information from Vigiconf concerning the topology's last modification "
"date. Therefore the topology has NOT been rebuilt."
msgstr ""
"Aucune information en provenance de Vigiconf concernant la date de "
"dernière modification de la topologie du parc. Par conséquent la "
"topologie N'A PAS été recalculée."

#: src/vigilo/correlator/actors/rule_dispatcher.py:122
msgid "Topology has been reloaded."
msgstr "L'arbre topologique a été reconstruit."

#: src/vigilo/correlator/actors/rule_dispatcher.py:235
#, python-format
msgid "Got item: %s"
msgstr "Réception d'un élément : %s"

#: src/vigilo/correlator/actors/rule_dispatcher.py:240
#, python-format
msgid "Skipping unrecognized item (%s)"
msgstr "Élément non-reconnu ignoré (%s)"

#: src/vigilo/correlator/actors/rule_dispatcher.py:249
msgid "The correlator is shutting down due to a previous error"
msgstr "Une erreur a été détectée qui va entraîner l'arrêt du corrélateur"

#: src/vigilo/correlator/actors/rule_dispatcher.py:302
#, python-format
msgid "Rule %(rule_name)s timed out"
msgstr "La règle %(rule_name)s a dépassé le délai d'exécution"

#: src/vigilo/correlator/actors/rule_dispatcher.py:418
#, python-format
msgid "Spawning for payload: %s"
msgstr "Traitement du message : %s"

#: src/vigilo/correlator/actors/rule_dispatcher.py:425
msgid "Received invalid XMPP item ID (None)"
msgstr "Réception d'un ID XMPP non valide (None)"

#: src/vigilo/correlator/actors/rule_dispatcher.py:435
#, python-format
msgid "Validation failure: %s"
msgstr "Échec de la validation : %s"

#: src/vigilo/correlator/actors/rule_dispatcher.py:475
msgid "Could not connect to the database server, make sure it is running"
msgstr ""
"Échec de la connexion à la base de données, veuillez vous assurer que le "
"serveur est démarré"

#: src/vigilo/correlator/actors/rule_dispatcher.py:583
#, python-format
msgid "Chat message to forward: '%s'"
msgstr "Message direct à transmettre : '%s'"

#: src/vigilo/correlator/actors/rule_dispatcher.py:595
msgid "Connection initialized"
msgstr "Connexion initialisée"

#: src/vigilo/correlator/actors/rule_dispatcher.py:605
#, python-format
msgid "Starting the queue manager with a delay of %f seconds."
msgstr "Démarrage du gestionnaire de file avec un délai de %f secondes."

#: src/vigilo/correlator/actors/rule_dispatcher.py:619
msgid "Connection lost"
msgstr "Connexion perdue"

#: src/vigilo/correlator/actors/rule_dispatcher.py:657
#, python-format
msgid ""
"Unable to forward one-to-one message to XMPP server (no connection "
"established). The message (%s) has been stored for later retransmission."
msgstr ""
"Impossible de transmettre le message one-to-one au serveur XMPP (pas de "
"connexion). Le message (%s) a été stocké en vue d'une retransmission "
"ultérieure."

#: src/vigilo/correlator/actors/rule_dispatcher.py:676
#, python-format
msgid ""
"Unable to forward the message (%(error)r), it has been stored for later "
"retransmission (%(xml_src)s)"
msgstr ""
"Impossible de transmettre le message (%(xml_src)s) à cause d'une erreur "
"(%(error)r); il a été stocké en vue d'une retransmission ultérieure."

#: src/vigilo/correlator/actors/rule_dispatcher.py:687
#, python-format
msgid "No destination node configured for messages of type '%s'. Skipping."
msgstr "Aucun nœud cible configuré pour les messages de type '%s'. Message ignoré."

#: src/vigilo/correlator/actors/rule_dispatcher.py:696
#, python-format
msgid ""
"Message from Socket impossible to forward (no connection to XMPP server),"
" the message is stored for later reemission (%s)"
msgstr ""
"Impossible de transmettre le message depuis le socket (pas de connexion "
"au serveur XMPP), le message a été stockée en vue d'une retransmission "
"ultérieure (%s)"

#: src/vigilo/correlator/actors/rule_runner.py:53
#, python-format
msgid "Rule runner: process begins for rule \"%s\""
msgstr "Rule runner : démarrage du processus pour la règle \"%s\""

#: src/vigilo/correlator/actors/rule_runner.py:54
#, python-format
msgid "Process id: %(pid)r | Parent id: %(ppid)r | Rule name: %(name)s"
msgstr ""
"Id du processus : %(pid)r | Id du processus parent: %(ppid)r | Nom de \"\n"
"\"la règle : %(name)s"

#: src/vigilo/correlator/actors/rule_runner.py:62
#, python-format
msgid "Rule runner: process ends for rule \"%s\""
msgstr "Rule runner : arrêt du processus pour la règle \"%s\""

#: src/vigilo/correlator/rules/example.py:23
#, python-format
msgid "id %(id)s payload %(payload)s"
msgstr "id %(id)s message %(payload)s"

#: src/vigilo/correlator/rules/hls_deps.py:101
#, python-format
msgid "Impacted HLS = %(affected_hls)r for (%(hostname)r, %(servicename)r)"
msgstr "HLS impactés par (%(hostname)r, %(servicename)r) = %(affected_hls)r"

#: src/vigilo/correlator/rules/priority.py:79
#, python-format
msgid ""
"The priority for the alert on (%(hostname)r, %(servicename)r) is "
"%(priority)r."
msgstr ""
"La priorité de l'événement corrélé causé par (%(hostname)r, "
"%(servicename)r) est %(priority)r."

#: src/vigilo/correlator/rules/priority.py:120
#, python-format
msgid ""
"The priority of the correlated event(s) %(updated)d has been updated, "
"based on information about the HLS %(hls)r"
msgstr ""
"La priorité des événements corrélés %(updated)d a été mise à jour, en "
"utilisant les informations concernant le SHN %(hls)r"

#: src/vigilo/correlator/rules/update_occurrences_count.py:88
#, python-format
msgid ""
"The state for '%(servicename)s' on '%(hostname)s' did not change or "
"indicates no problem whatsoever. The occurrences counter has not been "
"incremented. New state = %(statename)r vs %(previous_statename)r -- "
"Occurrences: %(occurrences)d"
msgstr ""
"L'état pour '%(servicename)s' sur '%(hostname)s' n'a pas changé ou "
"n'indique aucun problème. Le compteur d'occurrences n'a pas été "
"incrémenté. Nouvel état = %(statename)r contre %(previous_statename)r. "
"Occurrences : %(occurrences)d"

#: src/vigilo/correlator/rules/update_occurrences_count.py:105
#, python-format
msgid "Host: %(hostname)s, service: %(servicename)s, occurrences: %(occurrences)d"
msgstr ""
"Hôte : %(hostname)s, service : %(servicename)s, occurrences : "
"%(occurrences)d"

#~ msgid "Could not connect to memcached server, make sure it is running"
#~ msgstr ""
#~ "Connexion impossible avec le serveur "
#~ "memcached, veuillez vous assurer \"\n"
#~ "\"qu'il soit démarré."

